src/application/type/enum/input-type-enum.ts

```diff
@@ -2,5 +2,10 @@ export enum InputTypeEnum {
     Text = 'text',
     Email = 'email',
     Birthdate = 'birthdate',
-    Password = 'password'
+    Password = 'password',
+    Genre = 'genre',
```

Comment:
Pour rester dans la logique d'avoir l'enum en anglais : 
```suggestion
    Gender = 'gender',
```